cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		t.Fatalf("cmd.StdoutPipe: got %v, want nil", err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		t.Fatalf("cmd.Start: got %v, want nil", err)
+	}
+
+	in := bufio.NewScanner(stdout)
+
+	for i := 0; i < 1000; i++ {
+		if !in.Scan() {
+			t.Fatalf("Could not scan: got %v error, want no error", in.Err())
```

Comment:
just got %v not got %v error
just want nil, not want no error

Reply:
Fixed. I will get used to this format from now on :)!